CHANGES.rst

```diff
@@ -56,6 +56,8 @@ New Features
 
 - ``astropy.io.registry``
 
+  - New functions to unregister readers, writers, and identifiers. [#6216]
```

Comment:
Oops: https://github.com/astropy/astropy-tools/commit/127fffc1104c9e0f3404ce1e9185ca440e9cf192 😊 

Don't think we need a what's new, I don't think this is really exciting enough to qualify.